docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+
+We would be finished here, except that RDDs inside GeoTrellis are required to carry along a Metadata object that describes the context of the RDD.  This is created like so:
+```scala
+  val metadata = TileLayerMetadata(DoubleCellType,
+                                   ld,
+                                   ld.extent,
+                                   LatLng,
+                                   KeyBounds(SpatialKey(0,0),
+                                             SpatialKey(ld.layoutCols-1,
+                                                        ld.layoutRows-1)))
+```
+To combine the RDD and the metadata, we write `val resultRdd = ContextRDD(tileRdd, metadata)`.
+
+This resulting RDD is essentially the object of interest, though it is possible to write `resultRDD.stitch` to produce a single merged tile.  In the general case, however, the RDD may cover an area so large and in sufficient resolution that the result of stitching would be too large for working memory.  In that case, it would be necessary to write subtiles off to disk and stitch them using some external program like GDAL, or a custom application would need to be written to make use of the tiles that were produced.
+
+#### A Note on Running Example Code ####
+
+To run the above test code, it is necessary to have a compatible environment.  Spark code may experience failures if run solely in the Scala interpreter, as accessed through SBT's `console` command.  One way to ensure proper execution is to run in `spark-shell`, a Scala environment which provides a SparkContext made available through the variable `sc`.  Another way is to compile the application into a JAR file using `sbt assembly`, and to use `spark-submit`.  This latter option is the preferred method for Spark applications, in general, but for the purposes of trying out the provided code samples, `spark-shell` is the more sensible choice.  The use of `spark-submit` is beyond the scope of this documentation, but many resources are available on the internet for learning this tool.
+
+In either event, it will be necessary to install Spark in your local environment to run the code above.  Once that is done, you will need to clone the GeoTrellis repository from [Github](https://github.com/geotrellis/geotrellis).  From the root directory of that project, execute the provided `sbt` script.  Once SBT is loaded, the following sequence of commands can be executed:
+```scala
+  project util
+  assembly
+  project proj4
+  assembly
+  project vector
+  assembly
+  project raster
+  assembly
+  project spark
+  set test in assembly := {}
+  set assemblyMergeStrategy in assembly := { case "META-INF/MANIFEST.MF" => MergeStrategy.discard ; case _ => MergeStrategy.first }
+  assembly
+```
```

Comment:
No nee to run all those assemblies. Just the one will package all that stuff into the one jar:

```
project spark
 set test in assembly := {}
 set assemblyMergeStrategy in assembly := { case "META-INF/MANIFEST.MF" => MergeStrategy.discard ; case _ => MergeStrategy.first }
 assembly
```
and then
```bash
spark-shell --jars spark/target/scala-2.10/geotrellis-spark-assembly-[version].jar
```

The spark assembly will package up all the dependencies, including all the geotrellis subprojects that you listed.

I would recommend putting instead to package up spark-etl, and then you don't have to manually set sbt-assembly properties like merging:

```scala
project spark-etl
assembly
```

```bash
spark-shell --jars spark-etl/target/scala-2.10/geotrellis-spark-etl-assembly-[version].jar
```